src/js/actions/AppsActions.js

```diff
@@ -133,7 +133,7 @@ var AppsActions = {
         });
       });
   },
-  applySettingsOnApp: function (appId, settings) {
+  applySettingsOnApp: function (appId, settings, fromEdit = false) {
```

Comment:
I suggest renaming `fromEdit` to `isEditing`. I like [the convention](https://github.com/airbnb/javascript#23.3) of using `is`/`has` + _-ing_ form to indicate boolean states.